apple/testing/default_runner/macos_test_runner.template.sh

```diff
@@ -163,7 +175,7 @@ xcrun llvm-cov \
   -format lcov \
   "${lcov_args[@]}" \
   "$test_binary" \
-  @"$COVERAGE_MANIFEST" \
+  @"$llvm_coverage_manifest" \
```

Comment:
I thought that this would be enough to do the filtering, but it does not appear to be the case